Source/engine.cpp

```diff
@@ -2692,27 +2692,27 @@ void Cl2ApplyTrans(BYTE *p, BYTE *ttbl, int nCel)
 	BYTE *dst;
 	DWORD *pFrameTable;
 
-	/// ASSERT: assert(p != NULL);
-	/// ASSERT: assert(ttbl != NULL);
+	assert(p != NULL);
+	assert(ttbl != NULL);
 
 	for (i = 1; i <= nCel; i++) {
 		pFrameTable = (DWORD *)&p[4 * i];
 		dst = &p[pFrameTable[0] + 10];
 		nDataSize = CelGetFrameSize(p, i) - 10;
 		while (nDataSize) {
-			width = *dst++;
+			bBlock = *pImage++;

```

Comment:
where did this come from?

Reply:
Bad copy-paste when moving between branches :/